File: src/main/webapp/js/link.js
/*global Ext */
Ext.operon.tutorials.Link=Ext.extend(Ext.operon.tutorials.Resource,
{	
	constructor:function(tutorial,data)
	{	
		Ext.operon.tutorials.Link.superclass.constructor.call(this);
		this.link_id=data.id;
		//this.link_id=data.link_id;
		this.href=data.href;
		this.text=data.text;
	},
	
	render:function()
	{
		var buffer=[];
		buffer.push('<a href="'+this.href+'" target="_blank">'+this.text+'</a>');	
		return buffer.join('');
	}
});

File: src/main/java/org/operon/tutorials/Link.java
package org.operon.tutorials;

public class Link extends Resource
{
	protected String href="http://www.psu.edu";
	protected String text="click here";
	
	public Link(){}
	
	public Link(String linkid)
	{
		super(linkid);
	}

	public String getHref(){return this.href;}
	public void setHref(String href){this.href=href;}

	public String getText(){return this.text;}
	public void setText(String text){this.text=text;}
}
